charts/pega/charts/constellation/values.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Parameters for Constellation operation:
+# Docker repos and tag for image
+# dockerRepos: local/constellationui/service:v2
+# dockerRepos: meshbincam.pega.com:7000/constellationui/service
+dockerRepos: YOUR_IMAGE_HERE
+# Url (protocol+host) for Infinity cluster - used for REST calls
+# infinityUrl: http://1.2.3.4:5678/prweb/
+infinityUrl: YOUR_WEB_NODE_DOMAIN:30080/prweb/
```

Comment:
This should use the appropriate service from the parent chart. No URL should be necessary.

Reply:
Thank you @dcasavant we will not need this with the answer to line 31 in your above reply. post we will remove this.

Reply:
Refer above comment in(charts/pega/charts/constellation/templates/clln-deployment.yaml)  for more details .